pkg/osquery/table/launcher_info.go

```diff
@@ -98,6 +100,17 @@ func generateLauncherInfoTable(store types.GetterSetter) table.GenerateFunc {
 			return results, nil
 		}
 
+		if runtime.GOOS == "darwin" && agent.HardwareKeys() != nil && agent.HardwareKeys().Public() != nil {
+			jsonBytes, err := json.Marshal(agent.HardwareKeys())
+			if err != nil {
+				return nil, fmt.Errorf("marshalling hardware keys: %w", err)
+			}
+			results[0]["hardware_key"] = string(jsonBytes)
```

Comment:
this produces a json array of objects with uid & keys, need to see how K2 handles this

Reply:
This is changes the meaning of the column from prior _and_ makes it different per platform. That's going to be hard to manage inside k2. Instead, I would recommend we move to using `hardware_keys` (and `hardware_keys_source`) and be consistent consistent across platforms.

Reply:
addressed in https://github.com/kolide/launcher/pull/1647